src/routes/docs/tutorials/angular/step-5/+page.markdoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: tutorial
+title: Add navigation 
+description: Add navigation to your Angular applicating using Appwrite.
+step: 5
+---
+
+In our app we want to have a navigation bar that is always visible. We will add it to the `AppComponent` and use the `UserService` to show either:
+- a logout button if the user is logged in.
+- a login button if the user is not logged in.
+
+Update the `AppComponent` in `src/app/app.component.js`:
+
+```js
+import { Component } from '@angular/core';
+import { UserService } from './user.service';
+
+@Component({
```

Comment:
Should this be in multiple files for consistency? What is the style guide on Angular say?